lms/envs/bok_choy.env.json

```diff
@@ -103,6 +103,7 @@
     "PLATFORM_NAME": "edX",
     "REGISTRATION_EXTRA_FIELDS": {
         "level_of_education": "optional",
+        "dropdown": "optional",
```

Comment:
Does this do anything? As far as I can see there is no updated bok choy test.

Also this PR should ideally be backwards compatible with everyone's existing config, so there should not be any errors when `"dropdown"` is not present here.